ledger-core/virtual/integration/delegated_finished_test.go

```diff
@@ -73,234 +63,3 @@ func mockExecutor(t *testing.T, server *utils.Server, callMethod callMethodFunc,
 		nil,
 	)
 }
-
-// 1. Send CallRequest
-// 2. Change pulse in mocked executor
-// 4. Since we changed pulse during execution, we expect that VDelegatedRequestFinished will be sent
-// 5. Check that in VDelegatedRequestFinished new object state is stored
-func TestVirtual_SendDelegatedFinished_IfPulseChanged_WithSideEffect(t *testing.T) {
-	t.Log("C4935")
```

Comment:
There is a better test with the same scenario: `TestVirtual_Method_PulseChanged`, caseID = "C5211"